launch_testing_ros/test/examples/README.md

```diff
@@ -1,31 +1,23 @@
 # Examples
 
-## `talker_listener.test.py`
```

Comment:
Same comment as in https://github.com/ros2/launch/pull/312

Reply:
Done in https://github.com/ros2/launch_ros/pull/54/commits/7983a7b238648a181d06128b60406cccd802b125.